public/firefox.html

```diff
@@ -44,6 +44,7 @@ <h1 class="browser-name page-heading">{{ browser.name }}</h1>
             <li data-headset="htc_vive"><span>Save the <code>openvr_api.dll</code> file somewhere on your computer where the user running Firefox can read it (e.g., <code>c:\openvr\</code>).</span></li>
             <li data-headset="htc_vive"><span>In Firefox Nightly, navigate to <code>about:config</code>; change the value of <code>dom.vr.openvr.enabled</code> to <code>true</code> and <code>gfx.vr.openvr-runtime</code> to the full path of the <code>openvr_api.dll</code> file (e.g., <code>c:\openvr\openvr_api.dll</code>).</span></li>
             <li data-headset="oculus_rift"><span>Ensure that your Oculus Home settings allow for <a href="https://blog.mozvr.com/oculus-home-rift-cv1-webvr/#enablingunknownsources">Unknown Sources</a>.</span></li>
+            <li><span>On computers (especially laptops) with Nvidia chipsets, you may need to <a href="http://alteredqualia.com/texts/optimus/">enable the discrete GPU</a>.</span></li>
```

Comment:
Thanks for adding!

I'd add at least one of the methods of enabling it (I wish there were section `id` anchors on that page; it's a great reference, but it's a lot to read through).

Here's what I came up with (we could add children ordered lists later):

> For PCs (especially laptops) using Nvidia chipsets, there are several methods of <a href="https://alteredqualia.com/texts/optimus/">enabling the discrete GPU</a>. One way is from the <a href="https://www.nvidia.com/Download/Find.aspx">NVIDIA Control Panel</a>: Load <code>3D Settings</code> <code>&gt;</code> <code>Manage 3D Settings</code> <code>&gt;</code> <code>Program Settings</code> tab, and select <strong>Mozilla Firefox (firefox.exe)</strong> (with the blue Nightly icon) as the program to customize, and select <strong>&ldquo;High-performance NVIDIA processor"</strong> as the preferred graphics processor.


```html
For PCs (especially laptops) using Nvidia chipsets, there are several methods of <a href="https://alteredqualia.com/texts/optimus/">enabling the discrete GPU</a>. One way is from the <a href="https://www.nvidia.com/Download/Find.aspx">NVIDIA Control Panel</a>: Load <code>3D Settings</code> <code>&gt;</code> <code>Manage 3D Settings</code> <code>&gt;</code> <code>Program Settings</code> tab, and select <strong>Mozilla Firefox (firefox.exe)</strong> (with the blue Nightly icon) as the program to customize, and select <strong>&ldquo;High-performance NVIDIA processor"</strong> as the preferred graphics processor.
```